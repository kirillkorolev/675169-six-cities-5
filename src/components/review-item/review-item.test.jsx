import React from "react";
import renderer from "react-test-renderer";
import ReviewItem from "./review-item";

it(`Should ReviewItem render properly`, () => {
  const item = {
    photo: `img/avatar.jpg`,
    name: `Angelina`,
    rating: 4,
    text: `text text text`,
    date: 807926400000,
  };

  const tree = renderer
    .create(
        <ReviewItem
          item={item}
        />
    )
    .toJSON();
  expect(tree).toMatchSnapshot();
});
