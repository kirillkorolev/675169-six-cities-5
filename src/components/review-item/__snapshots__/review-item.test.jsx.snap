// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Should ReviewItem render properly 1`] = `
<li
  className="reviews__item"
>
  <div
    className="reviews__user user"
  >
    <div
      className="reviews__avatar-wrapper user__avatar-wrapper"
    >
      <img
        alt="Reviews avatar"
        className="reviews__avatar user__avatar"
        height="54"
        src="img/avatar.jpg"
        width="54"
      />
    </div>
    <span
      className="reviews__user-name"
    >
      Angelina
    </span>
  </div>
  <div
    className="reviews__info"
  >
    <div
      className="reviews__rating rating"
    >
      <div
        className="reviews__stars rating__stars"
      >
        <span
          style={
            Object {
              "width": "80%",
            }
          }
        />
        <span
          className="visually-hidden"
        >
          Rating
        </span>
      </div>
    </div>
    <p
      className="reviews__text"
    >
      text text text
    </p>
    <time
      className="reviews__time"
      dateTime="1995-08-09"
    >
      August 1995
    </time>
  </div>
</li>
`;
